draft-ietf-quic-transport.md

```diff
@@ -3948,8 +3993,33 @@ Fixed Bit:
 
 Spin Bit (S):
 
-: The sixth bit (0x20) of byte 0 is the Latency Spin Bit, set as described in
-  {{!SPIN=I-D.ietf-quic-spin-exp}}.
+: The third most significant bit (0x20) of byte 0 is the Latency Spin Bit.
```

Comment:
I think this would be easier to read if either all the text was here or if this referenced the section above.  Given it's a lot of text, I tend to think the latter is better.

Reply:
+1 to this